cosmos-gui/src/app.js

```diff
@@ -49,6 +51,10 @@ var scEndpoint = config.clusters.storage.endpoint;
 var ccPrivKey = config.clusters.computing.private_key;
 var ccUser = config.clusters.computing.user;
 var ccEndpoint = config.clusters.computing.endpoint;
+var httpsOptions = {
+    key: fs.readFileSync(config.private_ley_filekey),
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/6c6563e7fd8d41a80d9bbbd4ac8091ccfcfca831